env.sample

```diff
@@ -0,0 +1,3 @@
+PGUSER=btv
```

Comment:
mb shift away from btv as the standard?

guessing this is still included as non-env-variables here and there, so another branch for it in the future?

Reply:
Yeah we should but like you say, it's not fixed everywhere yet.